tests/ui/enum/enum-discrim-too-small.stderr

```diff
@@ -1,35 +1,99 @@
-error[E0600]: cannot apply unary operator `-` to type `u8`
+error[E0277]: cannot apply unary operator `-` to type `u8`
   --> $DIR/enum-discrim-too-small.rs:5:11
    |
 LL |     Cu8 = -23,
    |           ^^^ cannot apply unary operator `-`
    |
+   = help: the trait `Neg` is not implemented for `u8`
+   = help: the following other types implement trait `Neg`:
+             &f128
+             &f16
+             &f32
+             &f64
+             &i128
+             &i16
+             &i32
+             &i64
+           and 12 others
    = note: unsigned values cannot be negated
+help: you may have meant the unsigned equivalent of `u8` type
```

Comment:
I can't quite parse that sentence, maybe "you may have meant the wrapped-around value"?